.github/workflows/tests.yml

```diff
@@ -0,0 +1,43 @@
+name: concert tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9]
```

Comment:
Why drop 3.10?

Reply:
The python3.10 I removed until we migrate from nose. My fix depends on the import order, which is different for coverage, nosetest and nose.main() 🙄